src/routes/overviews/bufnmf.svx

```diff
@@ -0,0 +1,134 @@
+---
+title: Audio Decomposition using BufNMF
+blurb: |
+    Decompose an audio buffer into component parts using non-negative matrix factorization.
+tags:
+    - decomposition
```

Comment:
i think we could add some more tags here:

- nmf
- analysis
- machine learning

More tags helps the search too :)